.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,29 @@
+name: Main tag created workflow
+
+on:
+  push:
+    tags-ignore:
+    - 'newtonsoft-*'
+
+jobs:
+  build:
+    name: Github Actions Build
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v2
+      with:
+        dotnet-version: 7.*
+        include-prerelease: true
+    - name: Build Reason
+      run: echo ${{github.ref}} and ${{github.event_name}}
+    - name: Build with dotnet
+      run: find . -path ./template -prune -false -o -name '*.csproj' -execdir dotnet build --configuration Release --nologo {} \;
```

Comment:
I might have to change this or move the newtonsoft stuff out of the main repo otherwise this will find the csproj.

Reply:
I suppose it depends on how split out you want them to be. Right now it'll find all csprojs and build them; if that's an issue, we could look at using path filters in the GitHub Actions workflow to only build projects we're interested in.

I find it overkill, that project doesn't add much overhead now, but it's up to you.